lib/src/model/accessor.dart

```diff
@@ -17,57 +16,61 @@ import 'package:dartdoc/src/warnings.dart';
 
 /// Getters and setters.
 class Accessor extends ModelElement implements EnclosedElement {
-  GetterSetterCombo enclosingCombo;
+  /// The combo ([Field] or [TopLevelVariable] containing this accessor.
+  /// Initialized by the combo's constructor.
+  late final GetterSetterCombo enclosingCombo;
 
-  Accessor(PropertyAccessorElement element, Library library,
+  Accessor(PropertyAccessorElement? element, Library? library,
       PackageGraph packageGraph,
-      [ExecutableMember /*?*/ originalMember])
+      [ExecutableMember? originalMember])
       : super(element, library, packageGraph, originalMember);
 
   @override
-  CharacterLocation get characterLocation {
-    if (element.nameOffset < 0) {
-      assert(element.isSynthetic, 'Invalid offset for non-synthetic element');
+  CharacterLocation? get characterLocation {
+    if (element!.nameOffset < 0) {
+      assert(element!.isSynthetic, 'Invalid offset for non-synthetic element');
       // TODO(jcollins-g): switch to [element.nonSynthetic] after analyzer 1.8
       return enclosingCombo.characterLocation;
     }
     return super.characterLocation;
   }
 
   @override
-  PropertyAccessorElement get element => super.element;
+  PropertyAccessorElement? get element =>
```

Comment:
I might be wrong, but it looks like practically _every_ access of this field immediately null-checks it. Can it be made non-nullable?